.github/workflows/go.yml

```diff
@@ -106,6 +106,11 @@ jobs:
           make e2e-cleanup
           make e2e-setup
 
+      - name: Helm 3
+        uses: WyriHaximus/github-action-helm3@v2.0
+        with:
+          exec: helm install kruise https://github.com/openkruise/kruise/releases/download/v0.7.0/kruise-chart.tgz
```

Comment:
where does it be used?

Reply:
I am writing an e2e test that needs to use cloneset